### ISSUE TITLE
Cannot create dataset for SimpleShot application

### ISSUE THREAD

**Issue by KirillP2323** [nnstreamer/nntrainer#2127]
While running simpleshot app binary (generated from meson/ninja build), I encountered the following error:
```
$ build/Applications/SimpleShot/simpleshot_runner conv4 L2N tractor:turtle:squirrel:willow_tree:sunflower_20shot_seed:456_train.dat tractor:turtle:squirrel:willow_tree:sunflower_seed:456_test.dat Applications/SimpleShot
================================================================================
          Layer name          Layer type     Input dimension         Input layer
================================================================================
            backbone     backbone_tflite                                        
--------------------------------------------------------------------------------
              l2norm   preprocess_l2norm                                        
--------------------------------------------------------------------------------
                 knn        centroid_knn                                        
================================================================================
creating dataset failed
```

**Comment by jijoongmoon** [comment]
As you can see, there is input files in ```Applications/SimpleShot/tasks```. 
``` bash
$ ls
tractor-turtle-squirrel-willow_tree-sunflower_10shot_seed-456_train.dat  tractor-turtle-squirrel-willow_tree-sunflower_20shot_seed-456_train.dat  tractor-turtle-squirrel-willow_tree-sunflower_seed-456_test.dat  
tractor-turtle-squirrel-willow_tree-sunflower_1shot_seed-456_train.dat   tractor-turtle-squirrel-willow_tree-sunflower_5shot_seed-456_train.dat  
```

Could you try
``` bash
build/Applications/SimpleShot/simpleshot_runner conv4 L2N tractor-turtle-squirrel-willow_tree-sunflower_20shot_seed-456_train.dat tractor-turtle-squirrel-willow_tree-sunflower_seed-456_test.dat Applications/SimpleShot
```

**Comment by KirillP2323** [comment]
Thanks, that helped. The output is as follows now:
```
$ /usr/lib/nntrainer/bin/applications/simpleshot_runner conv4 L2N tractor-turtle-squirrel-willow_tree-sunflower_20shot_seed-456_train.dat tractor-turtle-squirrel-willow_tree-sunflower_seed-456_test.dat Applications/SimpleShot
================================================================================
          Layer name          Layer type     Input dimension         Input layer
================================================================================
            backbone     backbone_tflite                                        
--------------------------------------------------------------------------------
              l2norm   preprocess_l2norm                                        
--------------------------------------------------------------------------------
                 knn        centroid_knn                                        
================================================================================
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
#1/1 - Training Loss: 0 >> [ Accuracy: 73.3333% - Validation Loss : 0 ]
successfully ran
```
It's suspicious that training loss and validation loss are 0, is that a problem?

**Comment by jijoongmoon** [comment]
> It's suspicious that training loss and validation loss are 0, is that a problem?

AFIK, simpleshot is knn based algorithms and therefore there is not backpropagation and computation of loss. 
NNTrainer shows the loss here is that it is following just common routine to display output. Of course, we need to fix.

**Comment by KirillP2323** [comment]
Sounds good, thank you.
